src/strings/es.json

```diff
@@ -1600,5 +1600,9 @@
     "TypeOptionPluralBoxSet": "Colecciones",
     "ReleaseGroup": "Grupo de lanzamiento",
     "LabelSyncPlaySettingsSyncCorrectionHelp": "Activar la sincronización activa de la reproducción acelerando o moviéndose a la posición estimada. Desactivar esto en caso de cortes frecuentes en la reproducción.",
-    "Cursive": "Cursiva"
+    "Cursive": "Cursiva",
```

Comment:
Though I assume this is your native language, to avoid upsetting Weblate we should add these translations after the fact via https://translate.jellyfin.org so they're properly captured there. Not a blocker, but ideal.